bioio/writers/ome_zarr_writer.py

```diff
@@ -1,431 +1,478 @@
-                    chunk_dim_map[biob.dimensions.DimensionNames.SpatialY] = lasty
-                    chunk_dim_map[biob.dimensions.DimensionNames.SpatialX] = lastx
-
-                    chunks.append(
-                        dict(
-                            chunks=OmeZarrWriter._build_chunk_dims(
-                                chunk_dim_map=chunk_dim_map,
-                                dimension_order=dimension_order,
-                            ),
-                            compressor=default_compressor,
-                        )
-                    )
-                else:
-                    rescaley = int(math.ceil(chunk_dims[ydimindex] / scaler.downscale))
-                    rescalex = int(math.ceil(chunk_dims[xdimindex] / scaler.downscale))
-                    chunk_dims = tuple(list(chunk_dims[:-2]) + [rescaley, rescalex])
-
-                    chunks.append(
-                        dict(
-                            chunks=chunk_dims,
-                            compressor=default_compressor,
-                        )
-                    )
-
-        else:
-            scaler = None
-
-        # try to construct per-image metadata
-        ome_json = OmeZarrWriter.build_ome(
-            image_data.shape[zdimindex] if zdimindex > -1 else 1,
-            image_name,
-            channel_names=channel_names,  # type: ignore
-            channel_colors=channel_colors,  # type: ignore
-            # This can be slow if computed here.
-            # TODO: Rely on user to supply the per-channel min/max.
-            channel_minmax=[
-                (0.0, 1.0)
-                for i in range(image_data.shape[cdimindex] if cdimindex > -1 else 1)
+                    ],
+                }
+                for i in range(len(self.level_shapes))
             ],
-        )
-        # TODO user supplies units?
-        dim_to_axis = {
-            biob.dimensions.DimensionNames.Time: {
-                "name": "t",
-                "type": "time",
-                "unit": "millisecond",
-            },
-            biob.dimensions.DimensionNames.Channel: {"name": "c", "type": "channel"},
-            biob.dimensions.DimensionNames.SpatialZ: {
-                "name": "z",
-                "type": "space",
-                "unit": "micrometer",
-            },
-            biob.dimensions.DimensionNames.SpatialY: {
-                "name": "y",
-                "type": "space",
-                "unit": "micrometer",
-            },
-            biob.dimensions.DimensionNames.SpatialX: {
-                "name": "x",
-                "type": "space",
-                "unit": "micrometer",
-            },
         }
+        ome: Dict[str, Any] = {"multiscales": [multiscale], "version": "0.5"}
+        if channels:
+            ome["omero"] = {"version": "0.5", "channels": channels}
+        if creator:
+            ome["_creator"] = creator
+        self.root.attrs.update({"ome": ome})
```

Comment:
Is there a spot in the specification for `plates`? In general I think it would be good if we provide at least enough metadata here to grab out all of the "standard_metadata" fields we like to get. I think just `plates` for the well row/column is missing, but maybe timelapse interval too if the multiscales doesn't track the delta between timepoints.